lib/usd/ui/layerEditor/stageSelectorWidget.cpp

```diff
@@ -58,28 +61,29 @@ void StageSelectorWidget::setSessionState(SessionState* in_sessionState)
         &SessionState::stageListChangedSignal,
         this,
         &StageSelectorWidget::updateFromSessionState);
-    updateFromSessionState();
```

Comment:
Looked like we had a copy/paste error and ended up with multiple calls to updateFromSessionState